.github/workflows/test-chart-version.yaml

```diff
@@ -69,7 +68,7 @@ on:
           The deployment time-to-live, after which it will be automatically deleted.
           Examples: "30m", "1h", "2h", "1d".
         required: false
-        default: 1h
+        default: ""
```

Comment:
the 1h TTL  is actually needed for when deploying on infra teams AWS EKS cluster. they want the default to be 1h. Please undo this.